src/css/components/_control-bar.scss

```diff
@@ -12,7 +12,7 @@
 
 // Video has started playing
 .vjs-has-started .vjs-control-bar {
-  @include display-flex;
+  @include display-flex(flex-start, space-between);
```

Comment:
This change to the justification should be fine. For all sizes except tiny, the progress bar fills all available space, making the justification pretty much irrelevant. This seems like the right value for other use-cases (without a progress bar).

Reply:
I wonder if the issue is that the spacer has different settings applied to it in tiny mode:
```css
   .vjs-custom-control-spacer { @include flex(auto); }
   &.vjs-no-flex .vjs-custom-control-spacer { width: auto; }
```
I'd be more comfortable fiddling with those properties over this one, at first at least.

Reply:
Ah, sure.